packages/explat/src/anon.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * External dependencies
+ */
+import cookie from 'cookie';
+
+export const getAnonId = async (): Promise< string | null > => {
+	return cookie.parse( document.cookie ).tk_ai || null;
```

Comment:
I think we settled on the polling logic here?

Reply:
Right, we did. I added it in https://github.com/woocommerce/woocommerce-admin/pull/6669/commits/d6d73f9a9330c3083517c7c73ee83b4b57cabdb9